spartan/model/specgreedy/greedy.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
```

Comment:
This file may be duplicated with the one in Fraudar. We should invoke the one in Fraudar.